src/modules/infobox/components/styling/DrawingStylePopoverButton.vue

```diff
@@ -66,6 +77,7 @@ export default {
     methods: {
         /** Hides the popover container, can be called outside (by this component's parent) */
         hidePopover() {
+            console.log('debug: closing')
```

Comment:
either switch to a `log.debug` if this as some long term debug value, or remove it (no console.log allowed :wink: )